hack/dump.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
+
+set -e -o pipefail
+
+version=$(curl -L https://storage.googleapis.com/kubevirt-prow/devel/nightly/release/kubevirt/kubevirt/latest)
+binary=/tmp/kubevirt-${version}-dump
+if [ ! -f $binary ]; then 
+    curl -L https://storage.googleapis.com/kubevirt-prow/devel/nightly/release/kubevirt/kubevirt/${version}/testing/dump -o $binary
+    chmod 755 $binary
+fi
```

Comment:
can you pre-pull the dump binary in the `hack/functest.sh` script, or in some code path that ensures that it is present before the testsuite runs?

my thought here is that I don't want this debug logic to degrade and for us not to know about it. If the dump binary can't be retrieved, i want that to be in a critical code path that is always executed.